Source/WebKitLegacy/mac/DOM/DOMHTMLDocument.mm

```diff
@@ -60,14 +60,12 @@ - (DOMHTMLCollection *)scripts
 
 - (int)width
 {
-    WebCore::JSMainThreadNullState state;
-    return IMPL->width();
+    return NULL;
```

Comment:
This should also be return 0 I think. NULL is not an int.

Reply:
Done in [1f1978a](https://github.com/WebKit/WebKit/commit/1f1978a816967eb52e39cb17d27c6b03c576b9a3).